tox.ini

```diff
@@ -1,11 +1,13 @@
 [tox]
-envlist = py27-lint, py27-lint-imports, py27-lint-imports-include-list, py27-unit, qunit, mako-count, web-controller-line-count, py33-lint, py34-lint, py35-lint, validate-test-tools
+envlist = py27-lint, py27-lint-imports, py27-lint-imports-include-list, py27-unit, qunit, mako-count, web-controller-line-count, py33-lint, py34-lint, py35-lint, validate-test-tools, py27-lint-docstring-include-list, py27-lint-docstring
 skipsdist = True
 
 [testenv:py27-lint]
 commands = bash .ci/flake8_wrapper.sh
 whitelist_externals = bash
-deps = flake8
+deps =
+    flake8
+    flake8_docstrings==1.0.2
```

Comment:
s/flake8_docstrings/flake8-docstrings/

Also, should we pin this only on release branches, like flake8?